.storybook/preview.js

```diff
@@ -1,7 +1,7 @@
 /* global window */
 
 // NOTE: Use plain css for now. Switch to using Sass once someone get's it to load in storybook preview iframe.
-import '../src/styles/index.css'
```

Comment:
Do we still need index.css for other GDS users, like WP sites without sass support? Or can I remove it?

If we need it, I guess it should be built based on index.scss..?